src/transformer/descriptor/helper/helper.ts

```diff
@@ -70,6 +55,22 @@ export namespace TypescriptHelper {
         }
 
         return TypeChecker().typeToTypeNode(type);
+    }
+
+    export function getTypeParameterOwnerMock(d: ts.Declaration): ts.Declaration {
```

Comment:
to keep consistency this function should be PascalCase instead of camelCase

Reply:
okay :). Maybe one day we will have consistency across the project!!!